### DISCUSSION TITLE
How many types can we open any application

### DISCUSSION THREAD

**Discussion by vajrang-b** [vajrang-b/RPA-Developer-in-30-Days#11] [Q&A] [answered]
There are three ways to open application now you add all your answers

**Comment by Ulhasguttikar** [comment]
1) Open Application
2) Start Process
3) Attach Window

**Reply by vajrang-b** [reply]
attach window is wrong

**Comment by Manjushree15** [comment, marked as answer]
1. Open Application/Open Browser
2. Start Process

**Comment by TiwariRakesh** [comment]
open Application
Start Process

**Comment by mayajain2018** [comment]
1. Open Application
2. Start Process
3. Attach Window
